test/parallel/test-console-assing-undefined.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+// Should be above require, because code in require read console
+// what we are trying to avoid
+// set should be earlier than get
+
+global.console = undefined;
+
+// Initially, the `console` variable is `undefined`, since console will be
+// lazily loaded in the getter.
+
```

Comment:
Adding require common gives "'common' is assigned a value but never used". What would be appropriate resolution of the problem?